From akpm@osdl.org Fri Jul  8 23:55:58 2005
Message-Id: <200507090653.j696rgG3020431@shell0.pdx.osdl.net>
Subject: SPEAKUP: build fix
To: akpm@osdl.org, greg@kroah.com, mm-commits@vger.kernel.org
From: akpm@osdl.org
Date: Fri, 08 Jul 2005 23:53:05 -0700


From: Andrew Morton <akpm@osdl.org>

drivers/char/speakup/speakup.c:81: error: static declaration of 'errno' follows non-static declaration
include/linux/unistd.h:4: error: previous declaration of 'errno' was here

Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/char/speakup/speakup.c |    1 -
 1 files changed, 1 deletion(-)

--- gregkh-2.6.orig/drivers/char/speakup/speakup.c	2005-07-27 13:02:56.000000000 -0700
+++ gregkh-2.6/drivers/char/speakup/speakup.c	2005-07-27 13:41:06.000000000 -0700
@@ -78,7 +78,6 @@
 static special_func special_handler = NULL;
 special_func help_handler = NULL;
 
-static int errno;
 int synth_file_inuse = 0;
 short pitch_shift = 0, synth_flags = 0;
 static char buf[256];
